### ISSUE TITLE
Text Atoms & Markdown Content

### ISSUE THREAD

**Issue by georgiee** [satellytes/satellytes.com#251]
We have no way to create single headlines or paragraphs easily because everything is grouped in bigger concepts right now. Let's provide components. Those components are also the ones we need for the markdown ast approach anyway🙏

+ Use the i18n link for [generated anchors](https://github.com/satellytes/satellytes.com/blob/35f2ef45b16cb351696e5493f66bcf0c4101ab6d/src/components/legacy/markdown/custom-components.tsx#L44-L56)

**Comment by georgiee** [comment]
Things to consider when handling markdown: 

+ The markdown must go through the Gatsby pipeline so images can be processed properly
+ We use the already processed markdown and forward the ast presentation to our rehype processor defined in `MarkdownAst` via `<MarkdownAst htmlAst={markdown.htmlAst} />`
+ I have gone the way to define our very own prism parser in the past but I have been crawling back to `gatsby-remark-prismjs` I think because of the complexity to combine the existing markdown ast which things I do in the code parser (but I can't remember exactly what failed 😬)
+ It would be awesome to have support for custom tags in the markdown, something like `<my-component>` and I checked a few bits about that in the past. By disabling the XSS relevant checks in rehype this is possible but still limited the moment you nested those components. It would be nice to have something like that to output custom content because a code block opener like ` ``` ` or a simple image can't have any additional data while we would like to pass in attribution or things like attribution and such.

**Comment by georgiee** [comment]
* I think we still can't embed easily videos (youtube and friends). While there are plenty of community plugins like `gatsby-remark-embed-youtube` I think this would be better to implement ourselves to gain control over the privacy (pre-fetch preview image so users don't load directly from youtube etc). Embed iframe only after a click etc.
* We don't have an easy way to attribute images in the content. What's the markdown formtat for that we can use and map with rehype ?

**Comment by georgiee** [comment]
This is obsolete. We have worked on delivering many new components already.
